packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -83,20 +84,26 @@ class SimControl {
 
     final List<String> command = <String>[_xcrunPath, 'simctl', 'list', '--json', section.name];
     printTrace(command.join(' '));
-    final ProcessResult results = processManager.runSync(command);
+    final ProcessResult results = await processManager.run(command);
     if (results.exitCode != 0) {
       printError('Error executing simctl: ${results.exitCode}\n${results.stderr}');
       return <String, Map<String, dynamic>>{};
     }
-
-    return json.decode(results.stdout)[section.name];
+    try {
+      return json.decode(results.stdout)[section.name];
+    } on FormatException {
+      // We failed to parse the simctl output, or it returned junk.
```

Comment:
Maybe put some of this in a printTrace() or printError() depending on whether we can explain how to download the missing artifacts. Also if there are artifacts missing, could that be detected with a doctor validator?

Reply:
I don't think its our missing artifacts, but some sort of internal initialization that isn't done yet. Like, call me again I'm not ready? Just a theory, I didn't find anything searching for the string

Reply:
Updated to include printError and raw simctl output